pootle/apps/pootle_store/views.py

```diff
@@ -390,8 +391,7 @@ def get_units(request):
             # Since `extra()` has been used before, it's necessary to
             # explicitly request the `store__pootle_path` field. This is a
             # subtetly in Django's ORM.
-            uid_list = [u['id'] for u
-                        in step_queryset.values('id', 'store__pootle_path')]
+            uid_list = list(step_queryset.values_list('id', flat=True))
```

Comment:
its been like that for a while - and in a later PR all of this is replace with a simple values_list as none of it is necessary any more